ktxspec.adoc

```diff
@@ -656,11 +657,13 @@ inflated from the scheme prior to GPU sampling.
 
 [TIP]
 ====
-LZW-style lossless supercompression, e.g, scheme 2, is generally
-ineffective on the block-compressed data of GPU texture formats.
-It is best reserved for use with uncompressed texture formats or
-with block-compressed data that has been specially conditioned for
-LZW compression such as by _Rate-distortion Optimization_ <<RDO>>.
+Depending on the content of the image, LZW-style lossless
+supercompression, e.g, scheme 2, may be ineffective on the
+block-compressed data of GPU texture formats. Compressibility can
+be improved by specially conditioning the data for LZW compression
+such as with _Rate-distortion Optimization_ <<RDO>>. Such conditioning
+in an encoding detail which is transparent to the user of the
```

Comment:
/s/in/is

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/b4fa36e9730b8591d135ec5beef44079d1521d3f.